lib/pbkit/pbkit.h

```diff
@@ -21,6 +21,7 @@ extern "C"
 #endif
 
 #include <xboxkrnl/xboxkrnl.h>
+#include <stdbool.h>
```

Comment:
`stdbool.h` is not supposed to be used in C++, so this should get enclosed in a corresponding `#if`. Our CI trips over this problem atm as our version of PDCLib's `stdbool.h` is not C++ compatible (which it is allowed to be). I have reported this upstream nonetheless and a workaround has been added to make it more forgiving, but ofc we shouldn't rely on that.

Reply:
I've added a guard (`#if !defined(__cplusplus)`)